cypress/integration/jsEmployeesTableDOM.spec.js

```diff
@@ -0,0 +1,174 @@
+'use strict';
+
+Cypress.Commands.add('fillNewEmployeeForm',
+  (fullName, position, city, age, salary) => {
+    cy.get('[data-qa="name"]').type(fullName);
+    cy.get('[data-qa="position"]').type(position);
+    cy.get('[data-qa="office"]').select(city);
+    cy.get('[data-qa="age"]').type(age);
+    cy.get('[data-qa="salary"]').type(salary);
+    cy.get('button').contains('Save to table').click();
+  });
+
+Cypress.Commands.add('checkDataDoesntExist',
+  (fullName, salary) => {
+    cy.get('tbody').contains(fullName).should('not.exist');
+    cy.get('tbody').contains(salary).should('not.exist');
+  });
+
+Cypress.Commands.add('compareRowValuesAfterSort', (employeeName, rowValue) => {
+  cy.contains('tr', employeeName).find('td').should(($collection) => {
+    const tdValues = [];
+
+    for (let i = 0; i < 5; i++) {
+      tdValues.push($collection.get(i).innerText);
+    }
+
+    for (let i = 0; i < 5; i++) {
+      expect(tdValues[i]).to.equal(rowValue[i]);
+    }
+  });
+});
+
+const ColumnsNames = {
+  name: 1,
+  position: 2,
+  office: 3,
+  age: 4,
+  salary: 5,
+};
+
+const checkValuesSorted = function(value1, value2) {
+  if (isNaN(Number(value1))) {
+    if (value1.localeCompare(value2) === -1) {
+      return true;
+    }
+  }
+
+  if (Number(value2) >= Number(value1)) {
+    return true;
+  }
+
+  return false;
+};
+
+Cypress.Commands.add('checkValuesIsSorted', (columnName, direction) => {
+  cy.get(
+    `tr:nth-child(n) td:nth-child(${ColumnsNames[columnName.toLowerCase()]})`
+  ).then(($collection) => {
+    const columnValues = [...$collection].map((collection) =>
+      collection.innerText.replace('$', '').replace(',', ''));
+
+    for (let i = 1; i < columnValues.length; i++) {
+      if (direction === 'ASC') {
+        // eslint-disable-next-line max-len
```

Comment:
Linter helps us write cleaner code. fix linter errors instead of shutting it up :)